File: OMWAYServices/OMWAY/src/main/java/com/example/omway/dto/omwUser/DriverDto.java
package com.example.omway.dto.omwUser;

import lombok.Data;

import java.time.LocalDate;

@Data
public class DriverDto {
    private String driverCif;
    private String password;
    private String name;
    private String phone;
    private String email;
    private boolean state;

    //Driver
    private LocalDate dlExpirationDate;
}

File: OMWAYAppViews/OMWayApp/app/src/main/java/com/main/omwayapp/apirest/dto/omwayuser/DriverDto.kt
package com.main.omwayapp.apirest.dto.omwayuser

import android.os.Parcelable
import kotlinx.parcelize.Parcelize
import kotlinx.serialization.Contextual
import kotlinx.serialization.Serializable
import java.time.LocalDate

@Parcelize
@Serializable
data class DriverDto(
    val driverCif: String,
    val password: String,
    val name: String,
    val phone: String,
    val email:String,
    val state: Boolean,
    val dlExpirationDate: String
) : Parcelable